sceptre/config/reader.py

```diff
@@ -413,6 +414,16 @@ def _recursive_read(
 
         return config
 
+    def _write_debug_file(self, content):
+        with tempfile.NamedTemporaryFile(
+            mode="w", delete=False, prefix="rendered_"
+        ) as temp_file:
+            temp_file.write(content)
+            temp_file.flush()
+
+        print(f"Debug file location: {temp_file.name}")
```

Comment:
I think that...
1. We should only do this if the `--debug` flag has been passed to the CLI
2. We should log this at the debug level rather than using `print`

Reply:
@jfalkenstein I have a vague memory of you asking me to do something similar in the past, and I think I did. However, my thinking here is:

1. This message can only ever be printed when Sceptre already bombed out, dumped its stack trace, and gave up. As the caller, I would expect to always be told as much as I need to know here about what went wrong, and how we ended up here! The idea that you could get more info by adding --debug seems a bit counter-intuitive therefore, although I suppose many desperate users will try --debug anyway at this point.
2. We could log to debug level but that's again not really where a stack trace goes? So, it seems a bit unclean to me to have the stack trace message going to the screen and other import info going elsewhere to the logger.

However, if you really want it done this way, I'm happy to—just let me know how!

Reply:
@dboitnot A path to a temp file is being printed during after the stack trace and just before the program exits.

Reply:
@jfalkenstein  I belatedly agreed that making this work in debug mode only is correct. I forgot that Sceptre is normally hiding stack traces from us unless `--debug` is specified. So that's done.